src/config/dependencies.py

```
@@ -45,12 +47,12 @@ def get_jwt_auth_manager(settings: BaseAppSettings = Depends(get_settings)) -> J
     return JWTAuthManager(
         secret_key_access=settings.SECRET_KEY_ACCESS,
         secret_key_refresh=settings.SECRET_KEY_REFRESH,
-        algorithm=settings.JWT_SIGNING_ALGORITHM
+        algorithm=settings.JWT_SIGNING_ALGORITHM,
```

The JWTAuthManager is constructed using the algorithm value from settings. Ensure `settings.JWT_SIGNING_ALGORITHM` is always defined in all environments (developing/testing). If there's any chance it could be missing or None, consider validating settings early and raising a clear error so the app fails fast instead of encountering cryptic errors during token operations.